cmd/snap/cmd_sign.go

```diff
@@ -31,7 +31,10 @@ import (
 )
 
 var shortSignHelp = i18n.G("Sign an assertion")
-var longSignHelp = i18n.G(`Sign an assertion using the specified key, using the input for headers from a JSON mapping provided through stdin, the body of the assertion can be specified through a "body" pseudo-header.
+var longSignHelp = i18n.G(`
+The sign command signs an assertion using the specified key, using the
+input for headers from a JSON mapping provided through stdin, the body
```

Comment:
Was already there, but since we're here this would be slightly better:

"... stdin, the body ..." => "... stdin. The body ..."

Reply:
youch, missed that one